bedrock/firefox/templates/firefox/tracking-protection-tour.html

```diff
@@ -0,0 +1,103 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% extends "firefox/base-resp.html" %}
+
+{% block extra_meta %}<meta name="robots" content="noindex">{% endblock %}
+
+{#- This will appear as <meta property="og:title"> which can be used for social share -#}
+{% block page_og_title %}
+  {{ _('Choose the independent browser') }}
+{% endblock %}
+
+{#- This will appear as <meta property="og:description"> which can be used for social share -#}
+{% block page_og_desc %}
+  {{ _('Firefox is non-profit, non-corporate, non-compromised. Choosing Firefox isn’t just choosing a browser. It’s a vote for personal freedom.') }}
+{% endblock %}
+
+{#- Override <meta property="og:url"> for social share -#}
+{% block page_og_url %}{{ url('firefox') }}{% endblock %}
+
+{% block string_data %}
+data-panel1-title="{{ _('How Tracking Protection works') }}"
+data-panel1-text="{{ _('When the shield is visible, it means Firefox is actively blocking page elements that may track you.') }}"
+data-panel1-step="{{ _('1 of 3') }}"
+data-panel1-button="{{ _('Next') }}"
+
+data-panel3-title="{{ _('Want to make changes?') }}"
+data-panel3-text="{{ _('It’s easy to disable Tracking Protection for the website you’re on by clicking “Disable protection for this session”.') }}"
+data-panel3-text-alt="{{ _('It’s easy to enable Tracking Protection for the website you’re on by clicking “Enable protection”.') }}"
+data-panel3-step="{{ _('3 of 3') }}"
+data-panel3-button="{{ _('Got it!') }}"
+{% endblock %}
+
+{# Google Analytics should never be enabled on this page! #}
+{% block google_analytics %}{% endblock %}
+
+{% block site_css %}
+  {% stylesheet 'tracking-protection-tour' %}
+{% endblock %}
+
+{% block page_title_prefix %}{% endblock %}
+{% block page_title %}{{ _('Firefox Tracking Protection') }}{% endblock %}
+{% block body_id %}tracking-protection-tour{% endblock %}
+{% block body_class %}{% endblock %}
+
+{% block site_header %}{% endblock %}
+
+{% block content %}
+<main role="main">
+  <header>
+    <h1>{{ _('Firefox Tracking Protection') }}</h1>
+    <p>
+      {{ _('Want to turn off this feature?') }}
+      <a class="prefs-link" href="https://support.mozilla.org/kb/tracking-protection-pbm">{{ _('Visit Privacy Preferences') }}</a>
```

Comment:
I'd keep these two sentences in one string. I can totally see someone wanting to translate it as "To disable this feature visit ..."